<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1321</width>
    <height>591</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color : #1f1f21;
	color: #fff;
	font: 9pt &quot;휴먼모음T&quot;;
}
QPushButton{
    color:#e8e8e8;
    border-radius: 8px;
}
QTextEdit{
	color: #fff;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-10</y>
     <width>541</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
background-color : #2c2e31;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>0</y>
     <width>121</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : #2c2e31;
border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>0</y>
     <width>541</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 1px;
border-color: #rgb(0, 0, 0);
background-color : #2c2e31;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>121</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;휴먼모음T&quot;;
border-style: solid;
border-width: 1px;
border-color: #2c2e31;
background-color : #2c2e31;


</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
    </property>
    <property name="text">
     <string>모델 불러오기</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
	
border-color: #rgb(0, 0, 0);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
    </property>
    <property name="text">
     <string>이미지 검출</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
    </property>
    <property name="text">
     <string>텍스트 검출</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
    </property>
    <property name="text">
     <string>데이터셋 폴더선택</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{
	background-color:#505459;
	color: #fff;
}
QPushButton:pressed{
	border-bottom-color: rgb(0, 0, 0);
	border-right-color: rgb(0, 0, 0);
    background-color:rgb(162, 169, 180);
}
</string>
    </property>
    <property name="text">
     <string>데이터셋 적용폴더</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : #2c2e31;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
